### PR TITLE
[ST-404] Add "What is a Target?" section to Product Docs

### PR DESCRIPTION
**Pull request by mtsuji-cobalt**
### Added

- Section on what a target is and not

I used code format for the example URLs, to avoid bolding too much text.

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/_index.md:45]: In essence, the form URL is not different from subfolders in the first section like `https://example.com/blog`? Does it have to be highlighted in a separate section?

**Review by mahmoudadly-cobalt — APPROVED** [review]
The markdown code itself is fine. I have one content-related comment.

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Scans/_index.md:45]: @eugenerevzin what are your thoughts?

Comment (eugenerevzin) [reply content/en/Platform Deep Dive/Scans/_index.md:45]: @mtsuji-cobalt - Would it be difficult to make this it's own page, under the Scans section?  The content looks great, but marketing team wanted to see if we could do it standalone for easier linking.

Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Scans/_index.md:45]: @eugenerevzin we can create a new page but this section also does have a direct link via `platform-deep-dive/scans/#what-is-a-dast-target`

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/_index.md:17]: This statement will not be true if we publish it now. Should we keep this change until GA?

Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Scans/_index.md:17]: These documentation changes are part of the GA release, so I won't merge any of it until it's approved to do so.

**Review by mahmoudadly-cobalt — APPROVED** [review]

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (davidgm0) [inline content/en/Platform Deep Dive/Scans/_index.md:36]: Having headers breaks the enumeration (`3. ` next line), maybe we should take the enumeration away?

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (davidgm0) [inline content/en/Platform Deep Dive/Scans/_index.md:25]: Should this go into `targets.md` ?

File: content/en/Platform Deep Dive/Scans/targets.md
Comment (davidgm0) [inline content/en/Platform Deep Dive/Scans/targets.md:53]: 👇 I like the image! What do you think about having
`[Optional: Prefix] + Domain Name + [Optional: Path] = Unique DAST Target`

So that the order is the same? Maybe it gets a bit too long

**Review by davidgm0 — APPROVED** [review]
I think the content is quite clear. Left some minor comments, address as you prefer

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Scans/_index.md:25]: I think we can just link the Targets page within the Scans how-to, so that we don't break up the instructions.

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/_index.md:6]: My bad, I deleted an important syntax while resolving a conflict
```suggestion
  DAST Scanner - Web Application Vulnerability Scanning
  ---
```

File: content/en/Platform Deep Dive/Scans/_index.md
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/_index.md:30]: I updated the screenshots and missed this line. We no longer have a plus button.
So maybe something like
```suggestion
- To add a [target](/platform-deep-dive/scans/targets), click the Add Target button.
```

**Pull request merged**
